.github/workflows/compilers.yml

```diff
@@ -75,6 +75,12 @@ jobs:
             container: gcc-11
             configure_append: '--disable-shared optflags=-O2'
             # check: true
+          - key: default_cc
+            name: 'gcc-11 annocheck'
+            # value: 'gcc-11 -Wl,-z,now -Wl,-pie'
```

Comment:
I was try to set a proper and minimal gcc flags here. But so far working in progress. Your suggestion is welcome!